app/batch-vote/confirm-cart/index.tsx

```
@@ -0,0 +1,217 @@
+				totalProposal={String(items.length)}
+				totalDotSpent={totalDotSpent}
+				onConfirm={handleConfirmCart}
+				onClearCart={clearCartItems}
+			/>
+		</ThemedView>
+	);
+}
+
+interface BottomViewProps {
+	totalProposal: string,
+	totalDotSpent: Number,
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)